hbase-common/src/main/java/org/apache/hadoop/hbase/HBaseConfiguration.java

```diff
@@ -250,12 +250,12 @@ private static void applyClusterKeyToConf(Configuration conf, String key) throws
     conf.set(HConstants.ZOOKEEPER_ZNODE_PARENT, zkClusterKey.getZnodeParent());
     // Without the right registry, the above configs are useless. Also, we don't use setClass()
     // here because the ConnectionRegistry* classes are not resolvable from this module.
-    // This will be broken if ZkConnectionRegistry class gets renamed or moved. Is there a better
-    // way?
-    LOG.info("Overriding client registry implementation to {}",
-      HConstants.ZK_CONNECTION_REGISTRY_CLASS);
-    conf.set(HConstants.CLIENT_CONNECTION_REGISTRY_IMPL_CONF_KEY,
-      HConstants.ZK_CONNECTION_REGISTRY_CLASS);
+    // This will be broken if ZkConnectionRegistryInternal class gets renamed or moved. Is there
+    // a better way?
+    String ZKConnectionRegistryClass =
+      "org.apache.hadoop.hbase.client.ZKConnectionRegistryInternal";
+    LOG.info("Overriding client registry implementation to {}", ZKConnectionRegistryClass);
```

Comment:
It seems weird to reference an internal class in a log?

Reply:
We are logging like this currently. Presumably it was helpful during development, but I doubt it provides value now. I can remove it.

Reply:
I was more wondering if we should keep it as the public ZKConnectionRegistry, but yea also open to removing it. I just think we shouldn't info log something that is meant to be internal like the new class you created, it would be more confusing than anything

Reply:
Hmm you're right. I miss-understood the use of this method.